spec/spec.md

```diff
@@ -422,10 +425,14 @@ where:
 
 - `auth_token`: an opaque string representing a unique identifying token issued by the wallet endpoint to the dapp endpoint. The format and contents are an implementation detail of the wallet endpoint. The dapp endpoint can use this on future connections to reauthorize access to [privileged methods](#privileged-methods).
 - `accounts`: one or more value objects that represent the accounts to which this auth token corresponds. These objects hold the following properties:
-  - `address`: a base64-encoded address for this account
+  - `address`: a base64-encoded public key for this account. 
+  - `display_address`: (optional) the address for this account. The format of this string will depend on the chain, and is specified by the `display_address_format` field
+  - `display_address_format`: (optional) the format of the `display_address`.
+  - `public_key`: (optional) a base64-encoded public key for this account. This is an alias for `address` with more accurate terminology. If present, this param should be preferred over `address`.
```

Comment:
I'm not sure we should add this as an alias. There's no backward-compat concern at the moment (like with the `chain`/`cluster` parameter to `authorize`). If we do a breaking spec revision in the future, we can rename it then.

Reply:
removed. (and fixed the `display_*` params)

Reply:
We should add some commentary about `address`, in either the description or non-normative section. Could I lean on you @sdlaver for some help with the language here?

Reply:
How about something like:

> There are 3 parameters in the result that are derived from the unique identity of the account: `address`, `display_address`, and `display_address_format`. `address` contains the public key of the account (for whatever cryptographic system is used as the authority for the account). For example, in the Solana blockchain, this is the Ed25519 public key. `address` should always be encoded with base64, as this parameter is inteded for programmatic consumption rather than human consumption. In contrast, the `display_address` and `display_address_format` are representations of the account appropriate for human consumption. The contents of these two fields are chain-specific; for the Solana blockchain, `display_address_format` will be `base58`, and `display_address` will be the public key for the account, encoded with base58. Other chains may use different encodings and/or different methods for obtaining a human-consumable address.

As an aside, this doesn't address whether the response should include a version of `display_address` that is appropriate for programmatic consumption. Not every blockchain uses this (i.e. Solana does not), so it can always be addressed on a per-chain basis with spec extensions. If we find the need for it is common, we can always add it into future spec versions.